src/pow.cpp

```diff
@@ -61,7 +61,19 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
     // Retarget
     const arith_uint256 bnPowLimit = UintToArith256(params.powLimit);
     arith_uint256 bnNew;
-    bnNew.SetCompact(pindexLast->nBits);
+
+    // Special difficulty rule for Testnet4
+    if (params.fPowAllowMinDifficultyBlocks && params.hashGenesisBlock == uint256S("0x00000000da84f2bafbbc53dee25a72ae507ff4914b867c565be350b0da8bf043")) {
+        // Use the last non-special-min-difficulty-rules-block
+        const CBlockIndex* pindex = pindexLast;
+        const unsigned int pow_min{bnPowLimit.GetCompact()};
```

Comment:
e172a96c0781de2bbd69312905d2c16cc1745c2f: `const uint32_t`? (matching return type of `GetCompact()`)

Reply:
done